depth_first_traversal/DepthFirstTraversal.java

```diff
@@ -0,0 +1,38 @@
+class DepthFirstTraversal {
+
+    public static int[] arr;    // DFS traversal path
+    public static int k = 0;
```

Comment:
again and again the same problem.
@aviaryan you said I removed package test but still there.